source/documentation/3.9/rspec-core/file.Changelog.html

```diff
@@ -72,27 +73,24 @@ <h3>3.9.0 / 2019-10-07</h3>
 
 <p><a href="http://github.com/rspec/rspec-core/compare/v3.8.2...v3.9.0">Full Changelog</a></p>
 
-<p>Enhancements:</p>
-
-<ul>
-<li>Improve the handling of errors during loading support files, if a file
-errors before loading specs, RSpec will now skip loading the specs.
-(David Rodríguez, #2568)</li>
-<li>Add support for --example-matches to run examples by regular expression.
-(Sam Joseph, Matt Rider, @okothkongo1, #2586)</li>
-<li>Add <code>did_you_mean</code> suggestions for file names encountering a <code>LoadError</code>
-outside of examples. (@obromios, #2601)</li>
-<li>Add a minimalist quick fix style formatter, only outputs failures as
-<code>file:line:message</code>. (Romain Tartière, #2614)</li>
-<li>Convert string number values to integer when used for <code>RSpec::Configuration#fail_fast</code>
-(Viktor Fonic, #2634)</li>
-<li>Issue warning when invalid values are used for <code>RSpec::Configuration#fail_fast</code>
-(Viktor Fonic, #2634)</li>
-<li>Add support for running the Rake task in a clean environment.
-(Jon Rowe, #2632)</li>
-<li>Indent messages by there example group / example in the documentation formatter.
-(Samuel Williams, #2649)</li>
-</ul>
+<p>Enhancements:
+* Improve the handling of errors during loading support files, if a file
+  errors before loading specs, RSpec will now skip loading the specs.
+  (David Rodríguez, #2568)
+* Add support for --example-matches to run examples by regular expression.
+  (Sam Joseph, Matt Rider, @okothkongo1, #2586)
+* Add <code>did_you_mean</code> suggestions for file names encountering a <code>LoadError</code>
+  outside of examples. (@obromios, #2601)
+* Add a minimalist quick fix style formatter, only outputs failures as
+  <code>file:line:message</code>. (Romain Tartière, #2614)
+* Convert string number values to integer when used for <code>RSpec::Configuration#fail_fast</code>
+  (Viktor Fonic, #2634)
+* Issue warning when invalid values are used for <code>RSpec::Configuration#fail_fast</code>
+  (Viktor Fonic, #2634)
+* Add support for running the Rake task in a clean environment.
+  (Jon Rowe, #2632)
+* Indent messages by there example group / example in the documentation formatter.
+  (Samuel Williams, #2649)</p>
```

Comment:
Huh? Why have we lost this?

Reply:
See https://github.com/rspec/rspec.github.io/pull/138#discussion_r405415046 :)